runtime/doc/help.txt

```diff
@@ -179,14 +179,15 @@ Versions ~
 |deprecated.txt|  Deprecated items that have been or will be removed
 
 Other ~
-|terminal_emulator.txt|	Terminal buffers
-|term.txt|		Terminal UI
-|ui.txt|		Nvim UI protocol
-|channel.txt|		Nvim asynchronous IO
-|dev_style.txt|		Nvim style guide
-|job_control.txt|	Spawn and control multiple processes
-|luaref.txt|		Lua reference manual
-|luvref.txt|		Luv (|vim.loop|) reference manual
+|terminal_emulator.txt|	  Terminal buffers
+|term.txt|		  Terminal UI
+|ui.txt|		  Nvim UI protocol
+|channel.txt|		  Nvim asynchronous IO
+|dev_style.txt|		  Nvim style guide
+|job_control.txt|	  Spawn and control multiple processes
+|luaref.txt|		  Lua reference manual
+|luvref.txt|		  Luv (|vim.loop|) reference manual
+|supported_platforms.txt|   Supported platforms
```

Comment:
Could you also add `news.txt` here somewhere? (I forgot to.)

Reply:
I added it at the top.